google-cloud-datastore/src/test/resources/index.yaml

```diff
@@ -0,0 +1,32 @@
+indexes:
```

Comment:
How are we making use of this file? I couldn't find any script which deploy this file to datastore service.

Reply:
Sorry for not explaining this earlier. As in the previous comment, some of these tests/queries require indexes to run. This is a one time setup done by running `gcloud datastore indexes create index.yaml`. While this does not need to be run in a script repeatedly, I thought it would be useful to include the index file as a resource for developers who want to run this test in the future so they don't need to figure out the indexes themselves.

Reply:
Aah, that make sense, Can we add a comment on top of this file documenting this context ?